src/server/api/routes/civic/staking.js

```diff
@@ -1,9 +1,22 @@
 const { Router } = require('express');
 const { setPrivateCacheHeader } = require('../../middleware/cache');
-const { apiCivicLikerGetStaking } = require('../../util/api');
+const {
+  apiCivicLikerGetStaking,
+  apiCivicLikerGetStakingInfo,
+} = require('../../util/api');
 
 const router = Router();
 
+router.get('/civic/staking/info', async (req, res, next) => {
+  try {
+    const { data } = await apiCivicLikerGetStakingInfo(req);
+    res.set('Cache-Control', 'public, max-age=86400');
```

Comment:
please add stake while revalidate

Reply:
What do you mean?

Reply:
stale while revalidate, stupid auto correct...

Reply:
e.g. `Cache-Control: max-age= 86400, stale-while-revalidate=86400`

Reply:
Updated